packages/calypso-build/bin/copy-styles.js

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node
+
+/* eslint-disable import/no-nodejs-modules,no-console */
+
+// find all the packages
+const path = require( 'path' );
+const rcopy = require( 'recursive-copy' );
+
+const dir = process.cwd();
+
+const inputDir = path.join( dir, 'src' );
+const outputDirEsm = path.join( dir, 'dist', 'esm' );
+const outputDirCommon = path.join( dir, 'dist', 'cjs' );
+
+console.log( 'Copying styles %s', dir );
+
+const copyOptions = {
+	overwrite: true,
+	filter: '**/*.scss',
+	concurrency: 127,
+	debug: true,
+};
+
+rcopy( inputDir, outputDirEsm, copyOptions )
+	.then( results => {
+		console.log( 'copied %d files', results.length );
```

Comment:
i think we can drop all the logging here. it gets pretty noisy.

Reply:
Indeed:

![image](https://user-images.githubusercontent.com/96308/59697594-6a1c6300-91ee-11e9-809a-5c55d9153245.png)

Looks like `rcopy` is noisy enough on its own:

![image](https://user-images.githubusercontent.com/96308/59698149-60dfc600-91ef-11e9-8d86-573ae0f5067a.png)

Reply:
Disabling `rcopy`'s `debug` flag makes more of a difference:

![image](https://user-images.githubusercontent.com/96308/59698605-5540cf00-91f0-11e9-86a2-453e7dbce248.png)

Reply:
nice